chapter_did/chapter_did.Rmd

```diff
@@ -148,25 +155,43 @@ layout: true
 
 --
 
+.pull-left[
 ```{r, echo = F, out.width = "600px"}
 knitr::include_graphics("chapter_did_files/figure-html/nj_penn_map.png")
 ```
+]
 
+--
+
+.pull-right[
+<br>
+<br>
+Pennsylvania and New Jersey are ***very similar***: similar institutions, similar habits, similar consumers, similar incomes, similar weather, etc.
+]
 ---
 
 # Card and Krueger (1994): Methodology
 
 * Surveyed 410 fast-food establishments in New Jersey (NJ) and eastern Pennsylvania
 
+--
+
 * Timing:
+
+--
   - Survey before NJ MW increase: Feb/March 1992
+
+--
   - Survey after NJ MW increase: Nov/Dec 1992
-  
+
+--
+
 * What comparisons do you think they did?
 
-## Card and Krueger's data: `diffindiff` package
+--
 
-* Let's take a closer at their data
+.pull-left[
+Let's take a closer at their data
 ```{r, echo = TRUE, eval = TRUE}
   # install package that contains the cleaned data
 remotes::install_github("b-rodrigues/diffindiff")
```

Comment:
`remotes` output is too long and overwrites the right column. consider including that chunk twice, once run with hidden output (so no overwriting), and once with `eval=FALSE`